docs/en/development-guide.md

```diff
@@ -0,0 +1,82 @@
+# Development guide
+
+## Run your first build
+
+Follow [the "Run from source code" section](../../README.md) on README.
+
+## Setup environment for development
+
+### Setup Git hook
+
+Run the following shell command to setup Git hook. It runs format or lint codes before commiting changes.
+
+```shell
+$ make setup-hooks
+```
+
+### Setup VSCode config
+
+Save the following code as `.vscode/launch.json`.
+
+```json
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Start KHI Backend",
+            "type": "go",
+            "request": "launch",
+            "mode": "debug",
+            "program": "./cmd/kubernetes-history-inspector/",
+            "cwd": "${workspaceFolder}",
+            "args": [
+                "--host",
+                "127.0.0.1",
+                "--port",
+                "8080",
+                "--frontend-asset-folder",
+                "./dist",
+            ],
+            "dlvLoadConfig": {
+                "followPointers": true,
+                "maxVariableRecurse": 1,
+                "maxStringLen": 100000,
+                "maxArrayValues": 64,
+                "maxStructFields": -1
+            },
+        }
+    ], 
+}
+```
+
+You can run KHI with VSCode and features like break points are available with it.
+
+### Run frontend server for development
```

Comment:
Just checking, do we want to add Run backend section? Or, since we mention "You can run KHI with VSCode and features like break points are available with it." in the VS Code section, we consider it's obvious and skip that part?